docs/questions/sharing/visualizations/progress-bar.md

```diff
@@ -20,6 +20,14 @@ To create a progress bar you'll need:
 
 - A query that returns a single number, like "Sum of order quantity". Progress bar doesn't work with breakouts.
 
+  You can also use a query that returns several metrics in a single row:
```

Comment:
Realized that this also applies to progress - this is actually the more common format of results, I think

Reply:
Good catch.